src/Controllers/SubmitControllerCreate.php

```diff
@@ -88,4 +117,47 @@ function ($value)
 
 		return true;
 	}
+
+	/**
+	 * Check the CMS Version
+	 *
+	 * @return  false on failiure else the CMS Version
+	 *
+	 * @since   1.0
+	 */
+	private function checkCMSVersion($data)
+	{
+		$dotchecks = explode('.' $data);
+
+		// We ever use 2 dots and 3 parts in our CMS Version
+		if (count($dotchecks) != 3)
+		{
+			return false;
+		}
+
+		// The pulugin is installed since 3.5.0 other CMS Versions can't have this plugin installed
+		if (version_compare($data, '3.5.0', '<'))
```

Comment:
I wouldn't block 3.x at all.  Fake or not, there's a 3.2.1 install in the database.  Doing this blocks being able to distro the plugin as an optional thing for earlier releases.  Doubt anyone would add it, but it keeps the door open.